server/src/dependency/heed.rs

```diff
@@ -125,6 +125,7 @@ fn create_db() -> moved_storage_heed::Env {
 
     let env = unsafe {
         EnvOpenOptions::new()
+            .max_readers(20)
```

Comment:
I suppose it's just a heuristic? Or any specific reason for this being 5x of the RwLock reader max?